README.md

```diff
@@ -1,3 +1,101 @@
-# MoveSingleAPISwift
+# MoveSingleAPI SDK in Swift
 
-A description of this package.
+## Installation 💻
+You can use Swift Package Manager to integrate the library by adding the following dependency in the `Package.swift` file or by adding it directly within Xcode.
+
+`.package(url: "https://github.com/move-ai/MoveSingleAPISwift", from: "1.0.0")`
+
+## Usage 🤩
+Import the framework in your project:
+```swift
+import MoveSingleAPISwift
+```
+
+[Create an MoveSingleAPI key](https://move.ai) and add it to your configuration:
+```swift
+let move = Move(apiKey: "TOKEN")`
+```
+
+### Create simple Take
+```swift
+let take = try await move.createTake(
+    takeID: UUID().uuidString, 
+    videoURL: // URL to local video file
+)
+```
+
+### Upload Take
+Uploads the video file and the meta data and creates a new take in the system.
+```swift
+try await take.upload()
+```
+
+### Create Job
+Take has to be uploaded. This creates a new job that runs the video through the pipeline to generate the animation.
+```swift
+try await take.createJob()
+```
+
+### Get Results
+To check for a result, update the status of the current job of the Take, which is the last one created. As soon as the state of the is `finished` the outputfiles can be downloaded.
+```swift
+try await take.currentJob?.update()
+if take.currentJob?.state == .finished {
+    try await take.currentJob?.outputFiles[.preview]?.download()
+    let localPreviewURL = await take.currentJob?.outputFiles[.preview]?.localURL
+}
+```
+
+### Create Take with EnhancementData
+Enhancement Data is an optional Array of `EnhancementData` for every frame of the video.
+
+#### CameraDesignData
+`CameraDesignData` holds the internal size and position data of the camera. The information is extracted a given `CMSampleBuffer`.
```

Comment:
can we change The information is extracted a given... 
to 
The information is extracted from a given...